config/locales/views/en.yml

```diff
@@ -525,6 +525,11 @@ en:
   skins:
     confirm_delete:
       confirm_html: Are you sure you want to <strong><em>delete</em></strong> the skin "%{skin_title}"?
+  stats:
```

Comment:
Okay, yeah, here's the "one more" comment missing from my premature review:

Should these be attributes in the models locale file instead? I'm kind of shaky on my understanding of that, so I'll need someone else to weigh in, but I wanted to bring it up.

Reply:
I've tried to shift it to the models now.